infinispan-remote/src/main/java/org/hibernate/ogm/datastore/infinispanremote/configuration/impl/InfinispanRemoteConfiguration.java

```diff
@@ -31,6 +34,14 @@
 
 	private static final Log log = LoggerFactory.getLogger();
 
+	/*
+	 * The expected configuration value for some properties, an exception is thrown if the value is changed.
+	 */
+	private static final String[][] expectedValuesForHotRod = {
+			{ FORCE_RETURN_VALUES, "true" },
+			{ MARSHALLER, OgmProtoStreamMarshaller.class.getName() }
```

Comment:
ah, sorry I see that's exactly what you do. I got a bit confused as you don't have to set the Marshaller property, but this shouldn't be a problem.